lib/system/ansi_c.nim

```diff
@@ -180,6 +180,8 @@ proc c_printf*(frmt: cstring): cint {.
 
 proc c_fputs*(c: cstring, f: CFilePtr): cint {.
   importc: "fputs", header: "<stdio.h>", discardable.}
+proc c_fputc*(c: char, f: CFilePtr): cint {.
+  importc: "fputc", header: "<stdio.h>", discardable.}
```

Comment:
```suggestion
proc c_fputc*(c: cint, f: CFilePtr): cint {.
  importc: "fputc", header: "<stdio.h>", discardable.}
```
The C function seems to be `int fputc( int ch, FILE *stream );`

ref https://en.cppreference.com/w/c/io/fputc